boot.py

```diff
@@ -4,92 +4,92 @@
 import sublime_plugin
 
 # Please keep this list sorted (Edit -> Sort Lines)
-from .plugin.code_actions import LspCodeActionsCommand
-from .plugin.code_actions import LspRefactorCommand
-from .plugin.code_actions import LspSourceActionCommand
-from .plugin.code_lens import LspCodeLensCommand
-from .plugin.code_lens import LspToggleCodeLensesCommand
-from .plugin.color import LspColorPresentationCommand
-from .plugin.completion import LspCommitCompletionWithOppositeInsertMode
-from .plugin.completion import LspResolveDocsCommand
-from .plugin.completion import LspSelectCompletionCommand
-from .plugin.configuration import LspDisableLanguageServerGloballyCommand
-from .plugin.configuration import LspDisableLanguageServerInProjectCommand
-from .plugin.configuration import LspEnableLanguageServerGloballyCommand
-from .plugin.configuration import LspEnableLanguageServerInProjectCommand
-from .plugin.core.collections import DottedDict
+from .plugin.code_actions import LspCodeActionsCommand  # noqa: F401
+from .plugin.code_actions import LspRefactorCommand  # noqa: F401
+from .plugin.code_actions import LspSourceActionCommand  # noqa: F401
+from .plugin.code_lens import LspCodeLensCommand  # noqa: F401
+from .plugin.code_lens import LspToggleCodeLensesCommand  # noqa: F401
+from .plugin.color import LspColorPresentationCommand  # noqa: F401
+from .plugin.completion import LspCommitCompletionWithOppositeInsertMode  # noqa: F401
+from .plugin.completion import LspResolveDocsCommand  # noqa: F401
+from .plugin.completion import LspSelectCompletionCommand  # noqa: F401
+from .plugin.configuration import LspDisableLanguageServerGloballyCommand  # noqa: F401
+from .plugin.configuration import LspDisableLanguageServerInProjectCommand  # noqa: F401
+from .plugin.configuration import LspEnableLanguageServerGloballyCommand  # noqa: F401
+from .plugin.configuration import LspEnableLanguageServerInProjectCommand  # noqa: F401
+from .plugin.core.collections import DottedDict  # noqa: F401
 from .plugin.core.css import load as load_css
 from .plugin.core.open import opening_files
 from .plugin.core.panels import PanelName
-from .plugin.core.protocol import Error
-from .plugin.core.registry import LspNextDiagnosticCommand
-from .plugin.core.registry import LspOpenLocationCommand
-from .plugin.core.registry import LspPrevDiagnosticCommand
-from .plugin.core.registry import LspRestartServerCommand
+from .plugin.core.protocol import Error  # noqa: F401
```

Comment:
If not used in this module, `Error` import is probably obsolete.

Reply:
> If not used in this module, `Error` import is probably obsolete.

Maybe it's by design, who knows. People can `from LSP import Error`.